### ISSUE TITLE
Bootstrap modals support

### ISSUE THREAD

**Issue by catz** [jkusa/ember-cli-clipboard#180]
Please consider adding additional option "container"

From https://clipboardjs.com/ - Advanced Usage section.
For use in Bootstrap Modals or with any other library that changes the focus you'll want to set the focused element as the container value.

Without it component doesn't really copy anything when a button in a popup.

My quick dirty solution

```
import CopyButtonComponent from 'ember-cli-clipboard/components/copy-button';

export default class CopyButtonWithContainerComponent extends CopyButtonComponent {
  constructor() {
    super(...arguments);
  }

  _createClipboard() {
    const { clipboardText: text, delegateClickEvent } = this;
    const trigger =
      delegateClickEvent === false
        ? this._buttonElement
        : `#${this._buttonElement.id}`;

    return new window.ClipboardJS(trigger, {
      text: typeof text === 'function' ? text : undefined,
      container: this._buttonElement,
    });
  }
}
```

Thank you

**Comment by enna** [comment]
Up on it, I also have the issue. ^^ Could it be possible to handle the `container` option of `clipboard.js` ? :pray:

**Comment by drewctate** [comment]
👍 I also have this issue. Thanks for getting PR taken care of @enna. I'd love to see this support on NPM so we can use it!

**Comment by Monkeychip** [comment]
👍 I am also having this issue.

**Comment by drewctate** [comment]
As an interim solution until a release gets cut, I had luck with specifying a commit hash in my `package.json` file like so:
```
    "ember-cli-clipboard": "jkusa/ember-cli-clipboard#4af026ceb684dedd297d93b1da731156cf848963",
```

**Comment by jkusa** [comment]
Sorry for the delay on this. The `container` arg is now available in [v0.15.0](https://github.com/jkusa/ember-cli-clipboard/releases/tag/v0.15.0).
